citi-onboarding/celeiro-dos-bambas
# Feature(feature/branch-release)

## What I did

- I've switched the baseURL in the api-axios.js file to the the URL of the heroku

## How to test

In the folder "celeiro-de-bambas-server" 

yarn install
yarn develop

In the folder "celeiro-dos-bambas"

Switch to the branch "feature/branch-release" 
Go to src
Go to services
Go to api-axios.jg
Check if the baseURL was changed to " https://celeiro-bambas-server.herokuapp.com/ " of the Heroku 
yarn install
yarn start 


Test the things that you think are worth testing, even the ones that are not in this description
